
*********************************ERROR DUMP************************************
*logAction dump:
*	Function Name: 		createLogDir
*	Arguments: 		
*		1-cr.py-upgradeE7.csv-bench.csv-full-N6-e7-e7-600-E7_R02.04.045_EX24VDSL2_0022---2
*		./log/st_dhcpv6-csv-all-E7_R02-04-045_EX24VDSL2_0022-2015-11-25_142407.651000/2-2015-11-25_143009.360000
*	Keyword Arguments: 		
*-------------------------------------------------------------------------------
*ERROR MESSAGE:
*   [Error 2] The system cannot find the file specified: 'workspace'
*Traceback (most recent call last):
*  File "c:\workspace\idash\lib\runner.py", line 18, in inner
*    response = fun(*arg, **kwargs)
*  File "c:\workspace\idash\lib\runner.py", line 83, in createLogDir
*    os.chdir(dir)
*WindowsError: [Error 2] The system cannot find the file specified: 'workspace'
*-------------------------------------------------------------------------------
*globals=>
*	'CASE_MODE': set(['full', 'run', 'f', 'tear', 'nt', 'setup', 'norun', 'teardown', 's', 'r', 't', 'notear', 'nosetup', 'nr', 'ns', 'noteardown'])
*	'__author__': 'Sean Yu'
*	'__doc__': None
*	'__file__': 'c:\workspace\idash\lib\runner.pyc'
*	'__name__': 'runner'
*	'__package__': None
*	'array2html': <function array2html at 0x027DA370>
*	'case_runner': <function case_runner at 0x027DA1F0>
*	'concurrent': <function concurrent at 0x027DA2F0>
*	'createLogDir': <function inner at 0x027DA030>
*	'createLogger': <function inner at 0x027DBF70>
*	'errorlogger': None
*	'initDUT': <function inner at 0x027DA130>
*	'libpath': 'c:\workspace\idash\dut'
*	'logAction': <function logAction at 0x02763CB0>
*	'loop': <function loop at 0x027DA270>
*	'openDutLogfile': <function inner at 0x027DA0B0>
*	'os': <module 'os' from 'c:\Python27\lib\os.pyc'>
*	'pardir': 'c:\workspace\idash'
*	'releaseDUTs': <function releaseDUTs at 0x027DA2B0>
*	'run': <function inner at 0x027DA1B0>
*	'run1case': <function run1case at 0x027DA330>
*	'run_case_in_suite': <function run_case_in_suite at 0x027DA230>
*	'sub': 'dut'
*	'subfolder': ['lib', 'dut']
*	'sys': <module 'sys' (built-in)>
*	'unittest': <module 'unittest' from 'c:\Python27\lib\unittest\__init__.pyc'>
*
*locals =>
*	'datetime': <module 'datetime' (built-in)>
*	'dir': 'workspace'
*	'fullname': '1-cr.py-upgradeE7.csv-bench.csv-full-N6-e7-e7-600-E7_R02.04.-2015-11-25_143009.410000'
*	'listDir': <function listDir at 0x028DBE30>
*	'logpath': ['c:\', 'workspace', 'idash', 'test', 'log', 'st_dhcpv6-csv-all-E7_R02-04-045_EX24VDSL2_0022-2015-11-25_142407.651000', '2-2015-11-25_143009.360000']
*	'name': '1-cr.py-upgradeE7.csv-bench.csv-full-N6-e7-e7-600-E7_R02.04.'
*	'old_cwd': ['c:\', 'workspace', 'idash', 'test']
*	'old_logpath': './log/st_dhcpv6-csv-all-E7_R02-04-045_EX24VDSL2_0022-2015-11-25_142407.651000/2-2015-11-25_143009.360000'
*	'os': <module 'os' from 'c:\Python27\lib\os.pyc'>
*	'pat': '[^\w\-_.]'
*	're': <module 're' from 'c:\Python27\lib\re.pyc'>
*	'removelist': '\-_.'
*	'tm': '2015-11-25_143009.410000'
*
**********************************EREOR END*************************************


*********************************ERROR DUMP************************************
*logAction dump:
*	Function Name: 		createLogDir
*	Arguments: 		
*		1-cr.py-upgradeE7.csv-bench.csv-full-N6-e7-e7-600-E7_R02.04.050_EX24_0003-2.4.50.3
*		./log/st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_133151.213000/1-2015-11-26_133200.512000
*	Keyword Arguments: 		
*-------------------------------------------------------------------------------
*ERROR MESSAGE:
*   [Error 2] The system cannot find the file specified: 'idash'
*Traceback (most recent call last):
*  File "C:\workspace\idash\lib\runner.py", line 18, in inner
*    response = fun(*arg, **kwargs)
*  File "C:\workspace\idash\lib\runner.py", line 84, in createLogDir
*    os.chdir(dir)
*WindowsError: [Error 2] The system cannot find the file specified: 'idash'
*-------------------------------------------------------------------------------
*globals=>
*	'CASE_MODE': set(['full', 'run', 'f', 'tear', 'nt', 'setup', 'norun', 'teardown', 's', 'r', 't', 'notear', 'nosetup', 'nr', 'ns', 'noteardown'])
*	'__author__': 'Sean Yu'
*	'__doc__': None
*	'__file__': 'C:\workspace\idash\lib\runner.pyc'
*	'__name__': 'runner'
*	'__package__': None
*	'array2html': <function array2html at 0x02C53030>
*	'case_runner': <function case_runner at 0x02C50E70>
*	'concurrent': <function concurrent at 0x02C50F70>
*	'createLogDir': <function inner at 0x02C50CB0>
*	'createLogger': <function inner at 0x02C50C30>
*	'errorlogger': None
*	'initDUT': <function inner at 0x02C50DB0>
*	'libpath': 'C:\workspace\idash\dut'
*	'logAction': <function logAction at 0x02C505F0>
*	'loop': <function loop at 0x02C50EF0>
*	'openDutLogfile': <function inner at 0x02C50D30>
*	'os': <module 'os' from 'C:\Python27\lib\os.pyc'>
*	'pardir': 'C:\workspace\idash'
*	'releaseDUTs': <function releaseDUTs at 0x02C50F30>
*	'run': <function inner at 0x02C50E30>
*	'run1case': <function run1case at 0x02C50FB0>
*	'run_case_in_suite': <function run_case_in_suite at 0x02C50EB0>
*	'sub': 'dut'
*	'subfolder': ['lib', 'dut']
*	'sys': <module 'sys' (built-in)>
*	'unittest': <module 'unittest' from 'C:\Python27\lib\unittest\__init__.pyc'>
*
*locals =>
*	'datetime': <module 'datetime' (built-in)>
*	'dir': 'idash'
*	'fullname': '1-cr.py-upgradeE7.csv-bench.csv-full-N6-e7-e7-600-E7_R02.04.-2015-11-26_133209.875000'
*	'listDir': <function listDir at 0x02CBBF30>
*	'logpath': ['C:\', 'workspace', 'idash', 'test', 'log', 'st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_133151.213000', '1-2015-11-26_133200.512000']
*	'name': '1-cr.py-upgradeE7.csv-bench.csv-full-N6-e7-e7-600-E7_R02.04.'
*	'old_cwd': ['C:\', 'workspace', 'idash', 'test']
*	'old_logpath': './log/st_dhcpv6-csv-2-3-E7_R02-04-050_EX24_0003-2-4-50-3-2015-11-26_133151.213000/1-2015-11-26_133200.512000'
*	'os': <module 'os' from 'C:\Python27\lib\os.pyc'>
*	'pat': '[^\w\-_.]'
*	're': <module 're' from 'C:\Python27\lib\re.pyc'>
*	'removelist': '\-_.'
*	'tm': '2015-11-26_133209.875000'
*
**********************************EREOR END*************************************

